### PR TITLE
build: speed up .tar.gz by 10x

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

**Review by BenTheElder — APPROVED** [review]
/lgtm

File: build/root/WORKSPACE [hunk 1]
```diff
@@ -7,9 +7,9 @@ http_archive(
 
 http_archive(
     name = "io_kubernetes_build",
-    sha256 = "8e49ac066fbaadd475bd63762caa90f81cd1880eba4cc25faa93355ef5fa2739",
-    strip_prefix = "repo-infra-e26fc85d14a1d3dc25569831acc06919673c545a",
-    urls = ["https://github.com/kubernetes/repo-infra/archive/e26fc85d14a1d3dc25569831acc06919673c545a.tar.gz"],
+    sha256 = "709ecef69829b6e1ceea783907a7686932b48a8fa020bb73fbbade0fa9d90ea5",
+    strip_prefix = "repo-infra-17364b8203a958352051af0dfe5b444cd29adfd5",
+    urls = ["https://github.com/mikedanese/repo-infra/archive/17364b8203a958352051af0dfe5b444cd29adfd5.tar.gz"],
```
Comment (ixdy) [inline build/root/WORKSPACE]: we should probably hold on merging this until this gets merged

**Review by ixdy — DISMISSED** [review]
/lgtm cancel

File: build/root/WORKSPACE (hunk 1 above)
Comment (mikedanese) [reply build/root/WORKSPACE]: we should merge it then :+1:

**Pull request merged**